setup_env.sh

```diff
@@ -49,19 +57,24 @@ GetDefaultAllocatorOptions() {
 }
 
 GetDirectoryExtention() {
-  echo "_${EXTENSION_CC,,}_${EXTENSION_OS,,}_${1}"
+  echo "_${EXTENSION_CC}_${EXTENSION_OS}_${1}"
```

Comment:
Yeah, I think it's fine to remove case-normalization here.  In fact, I forget why we even enforced lowercase in the first place.